observe-agent.schema.json

```
@@ -0,0 +1,193 @@
+    },
+    "resource_attributes": {
+      "type": "object",
+      "additionalProperties": { "type": "string" },
+      "description": "Resource-level attributes such as service name or deployment environment to add to all telemetry data."
+    },
+    "health_check": {
+      "type": "object",
+      "description": "Configuration for the agent’s health check HTTP server.",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Enable or disable the health check endpoint."
+        },
+        "endpoint": {
+          "type": "string",
+          "description": "The address and port where the health check endpoint is hosted."
+        },
+        "path": {
+          "type": "string",
+          "description": "The HTTP path used to check agent health."
+        }
+      }
+    },
+    "internal_telemetry": {
+      "type": "object",
+      "description": "Telemetry about the agent's own internal operations.",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Enable or disable internal telemetry reporting."
+        },
+        "logs": {
+          "type": "object",
+          "description": "Configuration for internal telemetry logs.",
+          "properties": {
+            "enabled": {
+              "type": "boolean",
+              "description": "Enable or disable internal logging."
+            },
+            "level": {
+              "type": "string",
+              "description": "Log level for internal logs (e.g., ERROR, DEBUG)."
+            }
+          }
+        },
+        "metrics": {
+          "type": "object",
+          "description": "Configuration for internal telemetry metrics.",
+          "properties": {
+            "enabled": {
+              "type": "boolean",
+              "description": "Enable or disable internal metrics collection."
+            },
+            "host": {
+              "type": "string",
+              "description": "The host IP address where internal metrics are exposed."
+            },
+            "level": {
+              "type": "string",
+              "description": "Verbosity level of internal metrics."
+            },
+            "port": {
+              "type": "integer",
+              "description": "Port number where internal metrics are exposed."
+            }
+          }
+        }
+      }
+    },
+    "forwarding": {
+      "type": "object",
+      "description": "Settings for forwarding metrics and traces to Observe.",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Enable or disable telemetry forwarding."
+        },
+        "metrics": {
+          "type": "object",
+          "description": "Forwarding options for metrics.",
+          "properties": {
+            "output_format": {
+              "type": "string",
+              "description": "Format used for outputting metrics (e.g., otel)."
```

Can this use a pattern to validate the string? https://json-schema.org/draft/2020-12/json-schema-validation#name-pattern
ex:
```suggestion
              "description": "Format used for outputting metrics (e.g., otel).",
              "pattern": "otel|prometheus"
```
Not sure if I got the syntax right, but something like that.